guilded/channel.py

```diff
@@ -1,8 +1,11 @@
 from enum import Enum
 
 import guilded.abc
+
+from .message import Message
 from .utils import ISO8601
 
+
```

Comment:
The extra spaces seem less than needed to me. No added functionality, efficiency, or readability.

Reply:
Those are PEP8 changes. Aren't changes as those welcomed?